joseTOV/metamodel_eos.py

```diff
@@ -0,0 +1,521 @@
+        self.kappa_sym2 = self.kappa_NM2 - self.kappa_sat2
+        self.kappa_sym3 = self.kappa_NM3 - self.kappa_sat3
+        
+        # Kinetic energy: (t_sat is also called TFGsat in the margueron paper)
+        self.t_sat = 3 * utils.hbar ** 2 / (10 * utils.m) * (3 * jnp.pi ** 2 * self.nsat / 2) ** (2/3)
+        
+        # Potential energy 
+        # v_sat is defined in equations (22) - (26) in the Margueron et al. paper
+        # TODO: there are more terms here, perhaps check the other reference that Rahul shared?
+        v_sat_0 = self.E_sat -     self.t_sat * ( 1 +   self.kappa_sat +   self.kappa_sat2 +     self.kappa_sat3)
+        v_sat_1 =       -     self.t_sat * ( 2 + 5*self.kappa_sat + 8* self.kappa_sat2 + 11* self.kappa_sat3)
+        v_sat_2 = self.K_sat - 2 * self.t_sat * (-1 + 5*self.kappa_sat + 20*self.kappa_sat2 + 44* self.kappa_sat3)
+        v_sat_3 = self.Q_sat - 2 * self.t_sat * ( 4 - 5*self.kappa_sat + 40*self.kappa_sat2 + 220* self.kappa_sat3)
+        v_sat_4 = self.Z_sat - 8 * self.t_sat * (-7 + 5*self.kappa_sat - 10*self.kappa_sat2 + 110* self.kappa_sat3) 
+        
+        self.v_sat = jnp.array([v_sat_0, v_sat_1, v_sat_2, v_sat_3, v_sat_4])
+        
+        # v_sym2 is defined in equations (27) to (31) in the Margueron et al. paper
+        v_sym2_0 = self.E_sym -     self.t_sat * ( 2**(2/3)*( 1+  self.kappa_NM+   self.kappa_NM2+    self.kappa_NM3) - ( 1+  self.kappa_sat+   self.kappa_sat2+   self.kappa_sat3)  ) - v_nq[0]
+        v_sym2_1 = self.L_sym -     self.t_sat * ( 2**(2/3)*( 2+5*self.kappa_NM+8* self.kappa_NM2+ 11*self.kappa_NM3) - ( 2+5*self.kappa_sat+8* self.kappa_sat2+11*self.kappa_sat3)  ) - v_nq[1]
+        v_sym2_2 = self.K_sym - 2 * self.t_sat * ( 2**(2/3)*(-1+5*self.kappa_NM+20*self.kappa_NM2+ 44*self.kappa_NM3) - (-1+5*self.kappa_sat+20*self.kappa_sat2+44*self.kappa_sat3)  ) - v_nq[2]
+        v_sym2_3 = self.Q_sym - 2 * self.t_sat * ( 2**(2/3)*( 4-5*self.kappa_NM+40*self.kappa_NM2+ 220*self.kappa_NM3) - ( 4-5*self.kappa_sat+40*self.kappa_sat2+220*self.kappa_sat3) ) - v_nq[3]
+        v_sym2_4 = self.Z_sym - 8 * self.t_sat * ( 2**(2/3)*(-7+5*self.kappa_NM-10*self.kappa_NM2+ 110*self.kappa_NM3) - (-7+5*self.kappa_sat-10*self.kappa_sat2+110*self.kappa_sat3) ) - v_nq[4]
+        
+        self.v_sym2 = jnp.array([v_sym2_0, v_sym2_1, v_sym2_2, v_sym2_3, v_sym2_4])
+        
+                
+        ### TODO: empty crust not used for now
+        # if use_empty_crust:
+        #     ns_crust, ps_crust, es_crust = jnp.array([]), jnp.array([]), jnp.array([])
+        # else:
+        
+        # Load and preprocess the crust
+        ns_crust, ps_crust, es_crust = load_crust(crust)
+        mask = ns_crust <= max_n_crust
+        ns_crust, ps_crust, es_crust = ns_crust[mask], ps_crust[mask], es_crust[mask]
+        mu_lowest = (es_crust[0] + ps_crust[0]) / ns_crust[0]
+        
+        print("WARNING OVERRIDING FOR DEBUGGING")
+        mu_lowest = 930.1193490245807
```

Comment:
This is the value for `mu_lowest` as printed out by the DH crust used by Rahul's EOS code